src/composables/layout.ts

```diff
@@ -0,0 +1,24 @@
+import { ref } from 'vue'
+
+const isCartDrawerOpen = ref(false)
+
+const isMenuDrawerOpen = ref(false)
+
+export const usePublicLayout = () => {
+  const toggleCartDrawer = () => {
```

Comment:
Do we want to set a Type our methods?

Reply:
Sorry, I don't get it. Should I Type this `toggleCartDrawer` method?